hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -112,6 +112,18 @@
   public static final String
       DFS_CONTAINER_RATIS_LEADER_PENDING_BYTES_LIMIT_DEFAULT = "1GB";
 
+  public static final String DFS_RATIS_CLIENT_REQUEST_RETRY_POLICY =
+      "dfs.ratis.client.request.retry.policy";
+
+  public static final String DFS_RATIS_CLIENT_REQUEST_MAX_RETRIES_KEY =
+      "dfs.ratis.client.request.max.retries";
+  public static final int DFS_RATIS_CLIENT_REQUEST_MAX_RETRIES_DEFAULT = 180;
+  public static final String DFS_RATIS_CLIENT_REQUEST_RETRY_INTERVAL_KEY =
+      "dfs.ratis.client.request.retry.interval";
+  public static final TimeDuration
+      DFS_RATIS_CLIENT_REQUEST_RETRY_INTERVAL_DEFAULT =
+      TimeDuration.valueOf(1000, TimeUnit.MILLISECONDS);
+
```

Comment:
Can we add these configs to RatisClientConfig class? I would also suggest renaming some of the configs to match the existing configs like hdds.ratis.client.retrylimited.retry.interval or sth similar. This would make it easier to distinguish between the configs.

Reply:
Yeah，Sure.